canvasapi/canvas.py

```diff
@@ -738,6 +738,17 @@ def get_courses(self, **kwargs):
         )
 
     def get_current_user(self):
+        """
+        Return a details of the current user.
+
+        :calls: `GET /api/v1/users/:user_id \
+        <https://canvas.instructure.com/doc/api/users.html#method.current_user.show>`_
+
+        :param user: The object or ID of the current user.
+        :type user: :class:`cavasapi.user.User` or int
```

Comment:
`:param:` and `:type:` are used to document the arguments of the function. In the case of `get_current_user`, there are no arguments (besides `self`, which needs no introduction), so these lines aren't necessary. Please remove them.